redirects.js

```diff
@@ -29,6 +29,10 @@ const redirects = [
     from: '/docs/collecting-data/collecting-from-own-applications/scala-tracker/setup-2',
     to: '/docs/collecting-data/collecting-from-own-applications/scala-tracker/setup',
   },
+  {
```

Comment:
I think this does not redirect for subpages, could you test? (Redirects only work in a prod build, `yarn build` → check `build/` directory.)

If it does not work, we should try to get a wildcard redirect working, but I am not 100% sure that’s supported. Worst case, we keep this one only, and we are not too far from moving to Netlify where we can go to town with wildcards.

Reply:
It does not. I've implemented it as a javascript callback.